From 270688c1f8cfaed2e5c27da46151eac6eba2cfd8 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 16 Feb 2021 11:58:11 -0600
Subject: [PATCH] Revert "vmlinux.lds.h: Add PGO and AutoFDO input sections"

This reverts commit caf20def4cca3aadfed451cb665872cfe959c4ed.

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 include/asm-generic/vmlinux.lds.h | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/include/asm-generic/vmlinux.lds.h b/include/asm-generic/vmlinux.lds.h
index f65a924a75ab..4976f4d30f55 100644
--- a/include/asm-generic/vmlinux.lds.h
+++ b/include/asm-generic/vmlinux.lds.h
@@ -492,10 +492,7 @@
  */
 #define TEXT_TEXT							\
 		ALIGN_FUNCTION();					\
-		*(.text.hot .text.hot.*)				\
-		*(TEXT_MAIN .text.fixup)				\
-		*(.text.unlikely .text.unlikely.*)			\
-		*(.text.unknown .text.unknown.*)			\
+		*(.text.hot TEXT_MAIN .text.fixup .text.unlikely)	\
 		*(.text..refcount)					\
 		*(.ref.text)						\
 	MEM_KEEP(init.text*)						\
-- 
2.30.0

